sdks/python/apache_beam/examples/california_housing_clustering.py

```diff
@@ -40,17 +74,30 @@ def run(
   if not test_pipeline:
     pipeline = beam.Pipeline(options=pipeline_options)
 
-  data = (pipeline | read_csv(known_args.input))
+  data = pipeline | read_csv(known_args.input)
 
   features = ['longitude', 'latitude', 'median_income']
 
   housing_features = to_pcollection(data[features])
 
+  # 1. Calculate clustering centers and save model to persistent storage
   _ = (
       housing_features
       | beam.Map(lambda record: list(record))
-      | OnlineClustering(
-          OnlineKMeans, n_clusters=6, batch_size=256, cluster_args={})
+      | "Train clustering model" >> OnlineClustering(
+          OnlineKMeans,
+          n_clusters=6,
+          batch_size=256,
+          cluster_args={},
+          checkpoints_path='/tmp/checkpoints'))
```

Comment:
Can we make the checkpoints path configurable as a pipeline option? This will allow the pipeline to be run on remote runners (e.g. dataflow)